tests/e2e/tests/pilot/authn_policy_test.go

```diff
@@ -132,10 +140,21 @@ func TestAuthNJwt(t *testing.T) {
 		{dst: "b", src: "d", port: "8080", token: "testToken", expect: "200"},
 
 		{dst: "c", src: "a", port: "80", token: validJwtToken, expect: "200"},
+		{dst: "c", src: "a", port: "80", token: validJwt2Token, expect: "401"},
 		{dst: "c", src: "a", port: "8080", token: "invalidToken", expect: "401"},
 		{dst: "c", src: "b", port: "", token: "random", expect: "401"},
 		{dst: "c", src: "d", port: "80", token: validJwtToken, expect: "200"},
 
+		// Path "/health_check" is disabled for JWT authentication.
+		{dst: "c", src: "a", port: "80", path: "/health_check", token: "invalidToken", expect: "200"},
+		{dst: "c", src: "a", port: "80", path: "/health_check", token: validJwtToken, expect: "200"},
+		{dst: "c", src: "a", port: "80", path: "/health_check", token: validJwt2Token, expect: "200"},
+
+		// Path "/jwt2" is enabled only with JWT2.
```

Comment:
a bit confusing, may rename to something like ```only enabled only by issuer testing@secure.istio.io with valid jwt validJwt2Token``` ?

Reply:
Done